.gitlab/nersc-gitlab-ci.yml

```
@@ -0,0 +1,71 @@
+    - export CMAKE_BUILD_PARALLEL_LEVEL=32
+    - export ENV_CMAKE_OPTIONS=""
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_TESTS=ON"
+    - export SCRIPT_FILE=${SOURCE_DIR}/scripts/CTestRun.cmake
+    - export OMP_NUM_THREADS=4
+    - export OMP_PROC_BIND=spread
+    - export OMP_PLACES=cores
+
+EPYC-OMP-gcc:
+  stage: test
+  script:
+    - echo "Kokkos-OpenMP tests by $GITLAB_USER_LOGIN using gcc."
+    - module load PrgEnv-gnu gcc/12.2.0
+    - export BUILD_DIR=${SOURCE_DIR}/build_omp_gcc
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DCMAKE_CXX_COMPILER=g++"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ARCH_ZEN3=ON"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_OPENMP=ON"
+    - ctest -VV -D CDASH_MODEL="Nightly" -D CTEST_SITE="nersc" -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}" -S ${SCRIPT_FILE} -D CTEST_BUILD_NAME="AMD-EPYC-OpenMP-gcc/12.2"
+
+A100-CUDA-nvcc:
+  stage: test
+  script:
+    - echo "Kokkos-CUDA tests by $GITLAB_USER_LOGIN using nvcc."
+    - module load PrgEnv-gnu gcc/12.2.0
+    - export BUILD_DIR=${SOURCE_DIR}/build_cuda_nvcc
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DCMAKE_CXX_COMPILER=${SOURCE_DIR}/bin/nvcc_wrapper"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ARCH_AMPERE80=ON"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_CUDA=ON"
+    - ctest -VV -D CDASH_MODEL="Nightly" -D CTEST_SITE="nersc" -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}" -S ${SCRIPT_FILE} -D CTEST_BUILD_NAME="NVIDIA-A100-CUDA-nvcc/12.4"
+
+A100-CUDA-clang:
+  stage: test
+  script:
+    - echo "Kokkos-CUDA tests by $GITLAB_USER_LOGIN using clang."
+    - module load PrgEnv-llvm/1.0
+    - export BUILD_DIR=${SOURCE_DIR}/build_cuda_clang
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DCMAKE_CXX_COMPILER=clang++"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ARCH_AMPERE80=ON"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_CUDA=ON"
+    - ctest -VV -D CDASH_MODEL="Nightly" -D CTEST_SITE="nersc" -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}" -S scripts/CTestRun.cmake -D CTEST_BUILD_NAME="NVIDIA-A100-CUDA-clang/18"
+
+A100-OpenMPTarget-clang:
+  stage: test
+  script:
+    - echo "Kokkos-OpenMPTarget tests by $GITLAB_USER_LOGIN using clang."
+    - module load PrgEnv-llvm/1.0
+    - export BUILD_DIR=${SOURCE_DIR}/build_ompt_clang
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DCMAKE_CXX_COMPILER=clang++"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ARCH_AMPERE80=ON"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_OPENMPTARGET=ON"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DCMAKE_CXX_FLAGS='-Wno-unknown-cuda-version -Wno-pass-failed'"
+    - ctest -VV -D CDASH_MODEL="Nightly" -D CTEST_SITE="nersc" -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}" -S scripts/CTestRun.cmake -D CTEST_BUILD_NAME="NVIDIA-A100-OpenMPTarget-clang/18"
```

```suggestion
    - ctest -VV -D CDASH_MODEL="Nightly" -D CTEST_SITE="nersc" -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}" -S ${SCRIPT_FILE} -D CTEST_BUILD_NAME="NVIDIA-A100-OpenMPTarget-clang/18"
```
Minor change, for consistency with previous jobs